kuma/wiki/tests/test_views.py

```diff
@@ -1650,8 +1645,7 @@ def test_edit_midair_collisions(self, is_ajax=False, translate_locale=None):
             location_of_error = HTMLParser.HTMLParser().unescape(
                 resp.content[start_of_error: end_of_error]
             )
-        ok_(midair_collission_error in location_of_error,
-            "Midair collision message should appear")
+        assert midair_collission_error in location_of_error, "Midair collision message should appear"
```

Comment:
Since the line exceeds the PEP8 line-length limit and the assert error message doesn't really add any value, I would suggest dropping the error message:
```python
assert midair_collission_error in location_of_error
```